### DISCUSSION TITLE
Table Headers follow along with scrolling

### DISCUSSION THREAD

**Discussion by dmarquez-splunk** [grafana/grafana#65128] [Dashboards, Panels & Visualizations]
I think it would be nice to have a table's header stay fixed in a scrollable table view. Currently, the header is included in the scrolling and gets lost when scrolling down.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82754. Feel free to continue the discussion around this there. Thank you!
